src/AI/Muriel.java

```diff
@@ -571,9 +577,18 @@ private void queueUnitProductionActions(GameMap gameMap)
           for( Unit enemyUnit : unitLists.get(enemyCo) )
           {
             UnitMatchupAndMetaInfo umami = myUnitEffectMap.get(new UnitModelPair(counter, enemyUnit.model));
-            if( umami.costEffectivenessRatio >= COST_EFFECTIVENESS_MIN ) score++; // Plus one if it's worth building.
-            if( umami.costEffectivenessRatio >= COST_EFFECTIVENESS_HIGH ) score++; // An extra bump in score if they are very good vs this type.
-            if( umami.costEffectivenessRatio < COST_EFFECTIVENESS_MIN ) score--; // Discount if my counter is countered.
+            if( useDamageRatio )
+            {
+              if( umami.damageRatio >= 1.0 ) score++; // Plus one if it's worth building.
```

Comment:
If A has damageRatio >= 1 against B, it doesn't mean it can kill in one hit - it just means A can do at least as much damage to B as B can do to A. If the enemy builds heavy tanks, then heavy tanks or air units are gonna be the favored high-cost options. Then next turn, if Muriel isn't still rich, it'll be back to Mechs again, most likely.